app/javascript/Redux/Greetings/greetingsReducer.js

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+
+const GET_RANDOM_GREETING_SUCCESS = 'GET_RANDOM_GREETING_SUCCESS';
+const GET_RANDOM_GREETING_FAILURE = 'GET_RANDOM_GREETING_FAILURE';
+
+function getRandomGreetingSuccess(greeting) {
+  return {
+    type: GET_RANDOM_GREETING_SUCCESS,
+    greeting,
+  };
+}
+
+function getRandomGreetingFailure(error) {
+  return {
+    type: GET_RANDOM_GREETING_FAILURE,
+    error,
+  };
+}
+
+const initialState = {
+  greeting: '',
+  error: null,
+};
+
+export function getRandomGreeting() {
+  return async function (dispatch) {
+    try {
+      const response = await axios.get('/api');
```

Comment:
- [ Required ] I think it would be better if you use the right `URL` to fetch a random greeting from the database, according to the rails routes in your project the `URL`  is supposed to be `http://127.0.0.1:3000/api/greetings/random` or `http://localhost:3000/api/greetings/random` yet in this method you've used `/api` thus no greeting is fetched from the database.

Reply:
I beg to defer on this. Even with that axios.get('/api'); used, I am still able to fetch greetings as seen in this picture below:

Before implementing your suggestion:

![url](https://github.com/kennedyowusu/hello_rails-react/assets/32777408/96bdce39-f1fd-4802-bb1a-579d60517a0d)

Nevertheless, I have updated it according to your suggestion and it still works as seen in the pictures below:

After implementing your suggestion:

![2](https://github.com/kennedyowusu/hello_rails-react/assets/32777408/c90ac083-ceac-4a95-b794-f32a878bd90e)

The updated you asked for (which isn't necessary anywhere because the app still works without it)

![1](https://github.com/kennedyowusu/hello_rails-react/assets/32777408/fa3f9be9-4c9e-4b05-95c1-efac154ca819)

Reply:
Hi @kennedyowusu,

Let me reach out to you on Zoom to explain what I meant, actually, the application is supposed to be started using this command `./bin/dev` and not be run from the browser the way you've illustrated above.